src/main/java/seedu/jimi/model/FilteredListManager.java

```diff
@@ -297,6 +345,26 @@ public boolean run(ReadOnlyTask task) {
         }
         
     }
+    
+    /**
+     * Qualifier to check if task/event is overdue w.r.t. current time.
+     * @author zexuan
+     *
+     */
+    private class OverdueQualifier implements Qualifier {
+
+        @Override
+        public boolean run(ReadOnlyTask task) {
+            if(task instanceof Event) { //needed?
```

Comment:
i dono, u didnt want event to have completion leh
so im not sure if need, im thinking need tho